site/content/events/2016-seattle/_sponsors.txt

```diff
@@ -42,7 +42,7 @@ end
 <% @bsponsors = [
 { :image => "rei.png", :name => "REI", :link => "http://www.rei.com/"},
 { :image => "luxoft.png", :name => "Luxoft", :link => "http://www.luxoft.com/"},
-{ :image => "leankit100.jpg", :name => "Leankit", :link => "http://www.leankit.com/"}
+{ :image => "leankit100.png", :name => "Leankit", :link => "http://www.leankit.com/"}
```

Comment:
Just to be clear, you've indicated here that a `.png` should be shown instead of a `.jpg` - and said `.png` is in this commit - but a new `.jpg` is *also* in this commit (which isn't used?). Mistake or on purpose? :smile: